lib/api/user-profile/client.ts

```
@@ -35,15 +35,64 @@ export function useUserProfileClient() {
 
   return {
     getUserProfile: async (): Promise<UserProfile> => {
-      const apiClient = await getApiClient();
-      const response = await apiClient.get(USER_PROFILE_API.GET_PROFILE);
-      return response.data;
+      try {
+        const apiClient = await getApiClient();
+        const response = await apiClient.get(USER_PROFILE_API.GET_PROFILE);
+        return response.data;
+      } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)